_posts/2024-05-07-opentelemetry-metrics-visualization.md

```diff
@@ -0,0 +1,411 @@
+---
+layout: post
+title:  "Using OpenSearch to visualize metrics ingested with OpenTelemetry"
+authors:
+ - kschnitter
+date: 2024-07-16
+categories:
+ - technical-post
+meta_keywords: 
+meta_description: 
```

Comment:
@KarstenSchnitter - Please review and edit or add this meta to your blog: 

_meta_keywords:_ SAP Cloud Logging, Data Prepper ingestion, OpenTelemetry data models, OpenSearch Dashboards, observability plugin

_meta_description:_ Discover how SAP Cloud Logging created custom-built visualizations for analyzing metrics using the OpenTelemetry and Data Prepper ingestion tools in OpenSearch.

Reply:
This looks fine.